lms/djangoapps/verify_student/migrations/0014_set_copy_id_photo_from.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.db.models import Q
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """Add values for self referencing column 'copy_id_photo_from' in
+        table 'SoftwareSecurePhotoVerification'.
+
+        Set the 'copy_id_photo_from' column value to initial verification which
+        has the photo ID key.
+        """
+        verifications_without_photo_id = orm.SoftwareSecurePhotoVerification.objects.filter(
+            Q(photo_id_key__isnull=True) | Q(photo_id_key='')
+        )
+        for verification in verifications_without_photo_id:
+            # Get the initial verification of the user with the 'photo_id_key'
+            to_copy = orm.SoftwareSecurePhotoVerification.objects.filter(
+                Q(photo_id_key__isnull=False) | ~Q(photo_id_key=''),
+                user=verification.user,
+                status__in=["submitted", "approved"]
```

Comment:
@cpennington since there aren't much initial photo verification's with `photo_id_key` so I thinks its fine. What you think?